src/Symfony/Component/Lock/Store/PdoStore.php

```diff
@@ -215,6 +218,14 @@ public function exists(Key $key)
         return (bool) (\is_object($result) ? $result->fetchOne() : $stmt->fetchColumn());
     }
 
+    /**
+     *  Сhecking if it is possible to lazy create a table.
+     */
+    private function isLazyCreateTablePossible(object $connection): bool
+    {
+        return $this->lazyCreateTable && (!$connection->isTransactionActive() || \in_array($this->driver, ['pgsql', 'sqlite', 'sqlsrv'], true));
```

Comment:
if the options is false, you never create the table.

Before this change, even when the option was false you created a table

Reply:
This option was added to this PR. Before that, you could not choose by yourself whether to create the table in the background or not.